calendar/experiments/calendar/parent/ext-calendar-provider.js

```diff
@@ -62,13 +62,26 @@ class ExtCalendarProvider extends cal.provider.BaseClass {
     }
   }
 
+  setProperty(name, value) {
```

Comment:
Just returning a static value on getProperty has been fine in the past, but I'm not opposed to leaving this here.

Reply:
I tested it and it did cause visual glitches in 78 (the sidebar looses the read-only icon until the view is reloaded). That's the reason for the `setProperty` bit.